### ISSUE TITLE
Bug - Skip to content - focus remains at top of page in demos

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#7554]
**Describe the problem**
Followup based on comment https://github.com/patternfly/patternfly-react/pull/7520#issuecomment-1154088525

The skip to content component has a bug in demos where when you click it, focus remains at the top of the page/the page does not scroll to the main content.

**How do you reproduce the problem?**
1. Go to [skip to content React demo](https://www.patternfly.org/components/skip-to-content/react/basic/)
2. Press Tab to place focus on the Skip to content button, press Enter, then press Tab again. Notice that focus goes back to the skip button, and the page did not scroll to the main content.
    - If you click anywhere on the page first then follow the steps above, the page does scroll to the main content, but focus still remains at the top of the page when pressing Tab again
4.  Go to [bulk select table demo](https://www.patternfly.org/components/table/react-demos/bulk-select/#main-content-page-layout-default-nav) and repeat Step 2, and notice the same behavior.

**Expected behavior**
The skip to content in React demos should have the same behavior as seen in the [skip to content HTML demo](https://www.patternfly.org/v4/components/skip-to-content/html/basic/)

**Additional notes**

Possibly an issue with routing? Noticed when testing something via VO in Safari that the page does scroll to the linked content area, but then immediately returns to the top of the page.

**Comment by rebeccaalpert** [comment]
Still an issue in 6!
